build-tools/config/webpack/webpack.partial.conf.module.js

```diff
@@ -36,9 +34,6 @@ module.exports = ({ config, isDevelopment }) => webpackConfig => {
                 {
                   loader: 'sass-loader',
                   options: {
-                    importer: jsonImporter(),
-                    data: '@import "src/asset/style/utils.scss";',
```

Comment:
If this is removed then you have to import all the utils in every file you use them. Or did they change something that that is not required anymore. Otherwise it would be a downgrade if we remove this. This line safes quite a bit of time when developing css.

Reply:
These options are not valid any more. I couldn't find a replacement for them.

Reply:
They moved the node-sass options from the root to a special option called `sassOptions`. See https://github.com/webpack-contrib/sass-loader#sassoptions

Reply:
Moved it to the `sassOptions`